crates/frontend/src/pages/home.rs

```diff
@@ -50,6 +52,120 @@ fn gen_name_id(s0: &String, s1: &String, s2: &String) -> String {
     format!("{s0}::{s1}::{s2}")
 }
 
+#[component]
+fn all_context_view(config: Config) -> impl IntoView {
+    let Config {
+        contexts,
+        overrides,
+        default_configs,
+    } = config;
+    let rows = |k: &String, v: &Value, striked: bool| {
+        let default_iter = vec![(k.clone(), v.clone())];
+        v
+            .as_object()
+            .unwrap_or(&Map::from_iter(default_iter))
+            .iter()
+            .map(|(key, value)| {
+                let key = key.replace('"', "").trim().to_string();
```

Comment:
same here, possible to do JSON parse ?

Reply:
I will address this after my #221 PR gets merged, it has all the necessary logic to fix this